ui/src/app/feedback/_components/feedback-form.tsx

```diff
@@ -18,22 +18,30 @@ export const feedbackFormSchema = z.object({
     type: z.string(),
     name: z.optional(z.string()),
     email: z.string().email(),
-    message: z.string().min(15)
+    message: z
+        .string()
+        .min(15, 'Feedback must be at least 15 characters')
+        .max(500, 'Feedback must be 500 characters or less')
 });
 
 const FeedbackForm = ({ currentUser }: { currentUser: User }) => {
     const [showSuccessAlert, isShowSuccessAlert] = useState(false);
     const [showErrorAlert, isShowErrorAlert] = useState(false);
 
+    const MAX_LENGTH = 500;
```

Comment:
Can you move this to the .env file? You will need to prefix it with `NEXT_PUBLIC_` because it is being using in a client component. A more good name would be `NEXT_PUBLIC_FEEBACK_FORM_MAX_LENGTH`.

When you import from the .env, you will need to type-cast to a number.